pennylane/operation.py

```diff
@@ -152,7 +152,7 @@ class ObservableReturnTypes(Enum):
 
     def __repr__(self):
         """String representation of the return types."""
-        return self.value
+        return str(self.value)
```

Comment:
This change is just because codevec complained that __repr__ does not return a string...

Reply:
Weird 🤔